src/locales/en.json

```diff
@@ -1,10 +1,32 @@
 {
   "Blank": "Blank",
-  "Catalog":"Catalog",
+  "Catalog": "Catalog",
   "Click the backdrop to dismiss.": "Click the backdrop to dismiss.",
-  "Orders":"Orders",
-  "More":"More",
+  "Color": "Color",
+  "COPY": "COPY",
+  "in stock": "in stock",
+  "Orders": "Orders",
+  "Loading": "Loading",
+  "More": "More",
+  "OPEN": "OPEN",
```

Comment:
These strings should be sentence case always. Actual case of the label will be handled by the actual component.

Reply:
Done.